src/util/defaults.ml

```diff
@@ -135,6 +135,8 @@ let _ = ()
       ; reg Analyses "ana.specification"   "" "SV-COMP specification (path or string)"
       ; reg Analyses "ana.wp"              "false" "Weakest precondition feasibility analysis for SV-COMP violations"
       ; reg Analyses "ana.octapron.no_uints"    "false"  "Use OctApron without tracking unsigned integers."
+      ; reg Analyses "ana.arrayoob"        "false"        "Array out of bounds check"
+      ; reg Analyses "ana.octapron.no_floats"   "true"  "Use OctApron without tracking floats."
```

Comment:
Why is `ana.octapron.no_floats` being added here?

Reply:
This was probably caused by a weird merge order from our side.

Reply:
it should be resolved now